applications/ray/variables.tf

```diff
@@ -39,7 +39,7 @@ variable "ray_version" {
 variable "kubernetes_namespace" {
   type        = string
   description = "Kubernetes namespace where resources are deployed"
-  default     = "myray"
+  default     = "ml"
```

Comment:
can you actually make this `ai-on-gke` to be consistent with others?

Reply:
(not a blocking commnet)

Reply:
As I mentioned in #618, I set this to match the namespace name that we set in [workloads.tfvars](https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/ray/workloads.tfvars#L30) and updated this one so that they would be consistent.

Reply:
Also, the jupyter solution uses ml as well: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/jupyter/workloads.tfvars#L35

Reply:
And the rag solution uses rag: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/rag/workloads.tfvars#L27 & https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/rag/variables.tf#L42

Reply:
Where do you see the use of `ai-on-gke`?

Reply:
Discussed offline. We will follow up separately on the main branch.

Reply:
> I set this to match the namespace name that we set in [workloads.tfvars](https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/ray/workloads.tfvars#L30) and updated this one so that they would be consistent.

IMO the default namespace in the workloads.tfvars file should also be `ai-on-gke` instead of `ml`